docs/frontend-pr-analysis-workflow.md

```
@@ -179,6 +207,14 @@ Calculates coverage and posts PR comment per changed app:
 ### build
 Verifies code compiles/builds successfully per changed app.
 
+### i18n-check
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)